src/native/external/libunwind/src/aarch64/Ginit.c

```diff
@@ -46,12 +46,29 @@ unw_addr_space_t unw_local_addr_space = &local_addr_space;
 static inline void *
 uc_addr (unw_tdep_context_t *uc, int reg)
 {
-  if (reg >= UNW_AARCH64_X0 && reg < UNW_AARCH64_V0)
+#ifdef __FreeBSD__
+  if (reg >= UNW_AARCH64_X0 && reg <= UNW_AARCH64_X30)
```

Comment:
Does upstream need this predicate fix too?

https://github.com/libunwind/libunwind/blob/612ebb80368ef69f713de06eaa3d924403ba9288/src/aarch64/Ginit.c#L50

Reply:
Yes, I am waiting for CI, we didn't find the issue there, but JIT tests are failing.